Dockerfile

```diff
@@ -32,9 +33,9 @@ ENV     NODE_PATH /calypso/server:/calypso/client
 
 # Install base npm packages to take advantage of the docker cache
 COPY    ./package.json /calypso/package.json
-COPY    ./npm-shrinkwrap.json /calypso/npm-shrinkwrap.json
-# Sometimes "npm install" fails the first time when the cache is empty, so we retry once if it failed
-RUN     npm install --production || npm install --production
+COPY    ./yarn.lock /calypso/yarn.lock
+
+RUN     curl -o- -L https://yarnpkg.com/install.sh | bash && export PATH="$HOME/.yarn/bin:$PATH" && yarn install
```

Comment:
If we use something in both prod/dev, that makes it a plain dependency 😄 If you look at our package.json, babel should be under `dependencies` vs `devDependencies`